### DISCUSSION TITLE
Normal List Widget

### DISCUSSION THREAD

**Discussion by dillonhaughton** [beeware/toga#1443] [Q&A]
Is there a normal list widget I can used instead of the Detailed List Widget?

**Comment by freakboy3742** [comment]
What's a "normal" list? You can show list-like content with a Table widget (with a single data column), or a Select widget - do either of those meet your needs?

**Reply by dillonhaughton** [reply]
Really the Detailed List Widget without an Icon or Subtitle requirement. The simplest solution for my needs would be a default empty icon. The default icon is a gray box though.

**Reply by freakboy3742** [reply]
I'm still not sure what widget you're referring to. Can you provide a screenshot of an example of a "normal" list in another application?

**Comment by t-arn** [comment]
I think that Table is a poor replacement for a List Widget like Android's ListView because

- On Windows and macOS, you cannot control the width of the colums, they have a standard width (on Android, the column width  always matches the longest content)
- Table has a header that you might not want
- Table on Android seems to cause problems when the list is very long
- Table does not exist on iOS

**Comment by t-arn** [comment]
On mobile platforms, List Views are very common, but I don't know if there is something similar on the desktop platforms.
